### DISCUSSION TITLE
lhr unlocking does not start on 3060ti

### DISCUSSION THREAD

**Discussion by jiacnacho** [Lolliedieb/lolMiner-releases#802] [Q&A]
The program indicates that it starts unlocking, but it does not. I am using version 1.33, try to use version 1.34 and it does the same. In version 1.34, if I pre-overclock, it doesn't even apply lhr. Any ideas?

Im using windows 10, nvidia drivers 472.12 and msi 3060 ti ventus 2x v1

![image](https://user-images.githubusercontent.com/53543338/138020113-eed02f16-8311-4ccc-8def-23c3f812eca9.png)

![image](https://user-images.githubusercontent.com/53543338/138020244-38c8abaa-a8cf-4982-8bf1-e70d5ef6150d.png)

**Comment by Mestrezuin** [comment]
tente usar o comando disponivel na versao 1.3.4 --lhrtune offsya memoria e hynix? tem certeza da versao da sua placa ? pois v1 nao e lhr , pode tntar tambem o comando --lhrtune off

se ajudar por favor avise no forum
